common/components/componentsBySection/MyProjects/MyProjectCard.jsx

```diff
@@ -61,18 +61,40 @@ class MyProjectCard extends React.PureComponent<Props, State> {
   }
 
   _renderProjectStatus(): React$Node {
-    const header: string = this.state.isOwner
-      ? "Project Status"
-      : "Volunteer Status";
-    const status: string = this.state.isOwner
-      ? this.props.project.isApproved
-        ? "Published"
-        : "Unpublished"
-      : !this.props.project.isApproved
-      ? "Awaiting Approval"
-      : this.props.project.isUpForRenewal
-      ? "Expires on " + moment(this.props.project.projectedEndDate).format("l")
-      : "Active";
+
+    let header = ""
+
+    if (this.state.isOwner) {
+      header = "Project Status";
+    }
+
+    else {
+      header = "Volunteer Status";
+    }
```

Comment:
```suggestion
const header: string = this.state.isOwner ? "Project Status"  :  "Volunteer Status";
```
Using a conditional operator for a simple if/else value assignment is fine, and in many cases like this I think enhances readability.  It's just when we get to multi-pronged nested conditionals that readability becomes a real issue for this construction.